src/content/docs/kubernetes-pixie/kubernetes-integration/installation/reduce-ingest.mdx

```diff
@@ -1,114 +1,257 @@
 ---
-title: Reduce data ingest
+title: Reduce data ingested
 tags:
-  - Integrations
-  - Kubernetes integration
-  - Understand and use data
-metaDescription: 'New Relic''s Kubernetes integration: How to install and activate the integration, and what data is reported.'
+    - Integrations
+    - Kubernetes integration
+    - Understand and use data
+metaDescription: 'New Relic''s Kubernetes integration: How to reduce the data ingested.'
 redirects:
-  - /docs/integrations/kubernetes-integration/understand-use-data/find-use-your-kubernetes-data
-  - /docs/integrations/kubernetes-integration/understand-use-data/understand-use-data
-  - /docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm/#services
-freshnessValidatedDate: never
+    - /docs/integrations/kubernetes-integration/understand-use-data/find-use-your-kubernetes-data
+    - /docs/integrations/kubernetes-integration/understand-use-data/understand-use-data
+    - /docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm/#services
+freshnessValidatedDate: 2024-06-24
 ---
 
-## Reduce data ingest [#reducedataingest]
+If you've checked your data ingestion and you'd like to reduce the amount you have, you can do it through charts and through parameters.
 
-Our charts support setting an option to reduce the amount of data ingested at the cost of granularity. To enable it, set `global.lowDataMode` to `true` in the `nri-bundle` chart.
+You can reduce the data in your charts and works for all installation methods: the guided install, Helm, or Manifest. Our charts can help you adjust settings to reduce the data you take in, which may result in less detailed information. These settings allow you to focus on the most important data, which can help you identify the most important insights.
 
-`lowDataMode` affects four specific components of the `nri-bundle` chart outlined below.
+It's also possible to reduce data ingestion through parameters. However, the modification of the parameters will depend on the method used to install Kubernetes.
 
-### New Relic Infrastructure
+## Reduce data ingested by setting `global.lowDataMode` [#reduce-data-ingest]
 
-If `lowDataMode` is enabled, the default scrape interval changes from `15s` to `30s`.
-You can also specify a custom value for it using `config.interval`, which will take preference over `lowDataMode`.
+If you're looking to cut costs, you can modify the `global.lowDataMode` key value. This will cut down on the number of metrics sent. The modification of this key will depend on the method used to install Kubernetes:
 
-### Prometheus Agent Integration
+    * **Guided install** (New Relic CLI, Helm, Manifest). The`global.lowDataMode` key is enabled by default.
```

Comment:
Guided install (New Relic CLI, Helm, Manifest).  The lowDataMode mode is included by default in the command thanks to 3 parameters (NRI_CLI_LOW_DATA_MODE, NRI_CLI_PROMETHEUS_AGENT_LOW_DATA_MODE and NRI_CLI_LOGGING_LOW_DATA_MODE).

Reply:
Done.